packages/devtools/test/utils_test.dart

```diff
@@ -125,5 +125,158 @@ void main() {
         equals('[1 ms - 8 ms]'),
       );
     });
+
+    test('longestFittingSubstring', () {
+      const str = 'ComponentElement.performRebuild';
```

Comment:
test with some non-ascii characters as well.
For example: Ԫ
You can include non-ascii characters directly in Dart code.

Reply:
done.